CITATION.cff

```diff
@@ -0,0 +1,57 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+
+# Project information
+authors:
+  - name: "National Optical Astronomy Observatories"
+  - name: "The IRAF community"
+title: IRAF
+type: software
+abstract: "Image Reduction and Analysis Facility "
+repository-code: "https://github.com/iraf-community/iraf"
+url: "https://iraf-community.github.io"
+license-url: "https://github.com/iraf-community/iraf/blob/main/COPYRIGHT"
+
+# Version information
+version: "2.17"
+doi: "10.5281/zenodo.5816744"
+data-released: "2022-01-04"
+
+# Associated papers
+preferred-citation:
+  authors:
+    - family-names: Tody
+      given-names: Doug
+  title: "The IRAF Data Reduction and Analysis System"
+  type: article
+  issue: 627
+  journal: "Proc. SPIE Instrumentation in Astronomy VI"
+  year: 1986
+  pages: 733
+  doi: "10.1117/12.968154"
+  editors:
+    - family-name: Crawford
+      given-names: "D.L."
+  url: "https://www.spiedigitallibrary.org/conference-proceedings-of-spie/0627/1/The-Iraf-Data-Reduction-And-Analysis-System/10.1117/12.968154.short"
```

Comment:
Please make this the paper's URL identifier and add the ADS identifier as `type: other`; see my comment in https://github.com/citation-file-format/citation-file-format/issues/436.